manifests/i/icedterminal/ffmpeg/4.4.0.20210901/icedterminal.ffmpeg.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: icedterminal.ffmpeg
+PackageVersion: 4.4.0.20210901
+PackageLocale: en-US
+Publisher: icedterminal
+PublisherUrl: https://icedterminal.me
+#PublisherSupportUrl: 
+#PrivacyUrl: 
+Author: FFmpeg team
+PackageName: FFmpeg Essentials
+PackageUrl: https://www.ffmpeg.org/
+License: GPL v2 or later
+LicenseUrl: https://www.ffmpeg.org/legal.html
+Copyright: (C) FFmpeg team
+CopyrightUrl: https://www.ffmpeg.org/legal.html
+ShortDescription: Multimedia command lines tools
+Description: FFmpeg is an open source command line multimedia framework to encode, decode, transcode, convert etc. a wide array of audio and video formats.
+Moniker: ffmpeg
+Tags:
+- ffmpeg
+- transcode
+- encode
+- decode
+- convert
+- multimedia
+- video
+- audio
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```

Comment:
```suggestion
PackageIdentifier: icedterminal.ffmpeg
PackageVersion: 4.4.0.20210901
PackageLocale: en-US
Publisher: icedterminal
PublisherUrl: http://icedterminal.me
PublisherSupportUrl: https://github.com/icedterminal/ffmpeg-installer#issue-reporting
Author: icedterminal
PackageName: FFmpeg Essentials
PackageUrl: https://github.com/icedterminal/ffmpeg-installer
License: GPL v3
LicenseUrl: https://github.com/icedterminal/ffmpeg-installer/blob/main/LICENSE
Copyright: Copyright (c) icedterminal.
CopyrightUrl: https://github.com/icedterminal/ffmpeg-installer/blob/main/LICENSE
ShortDescription: An unofficial command line tool to convert multimedia files betwwen formats.
Description: An unofficial command line tool to convert multimedia files betwwen formats.
Moniker: ffmpeg-unofficial
Tags:
- ffmpeg
- transcode
- encode
- decode
- convert
- multimedia
- video
- audio
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```

Reply:
> An unofficial command line tool

I disagree. This is the official ffmpeg program as built from source.

> FFmpeg only provides source code.

FFmpeg team themselves do not provide builds. Your respective OS package manager team builds it or you do. EX: Ubuntu takes from ffmpeg source: https://launchpad.net/ubuntu/+source/ffmpeg as authored by the "Debian Multimedia Team" https://launchpad.net/~debian-multimedia

Reply:
>I disagree. This is the official ffmpeg program as built from source.

If that's the case, I would have expected the Installer URL to come from one of these distributions from the official website: https://www.ffmpeg.org/download.html#build-windows.

>FFmpeg team themselves do not provide builds. Your respective OS package manager team builds it or you do. EX: Ubuntu takes from ffmpeg source: https://launchpad.net/ubuntu/+source/ffmpeg as authored by the "Debian Multimedia Team" https://launchpad.net/~debian-multimedia

Lots of people have either repackaged installers into an `.exe` or `.msi` installer or uploaded the `.exe` file that was inside of the `.zip` file since Windows Package Manager (WinGet) doesn't support `.zip` files yet.

This is not allowed as we expect the installers to come from the publisher's website. In the case of FFmpeg, where the publisher doesn't build official builds, Gyan or BtbN builds can be used, but we're still blocked on `.zip` support for Windows Package Manager.

Reply:
> In the case of FFmpeg, where the developers don't build official builds for any OS, Gyan or BtbN builds can be used, but we're still blocked on .zip support for Windows Package Manager (WinGet). I think it would be best to wait for .zip support before we proceed further.

I see. I appreciate the insight. If you would rather wait, feel free to close this. 

> ItzLevvie suggested changes 12 hours ago

Again, for some reason when I click "View changes", I am shown my own commit with no suggested changes.

https://user-images.githubusercontent.com/4755734/133005371-5b4f6622-33c8-4b27-99ca-dd166c727108.mp4